packages/labs/ssr/package.json

```diff
@@ -195,7 +195,6 @@
     "@types/koa-cors": "*",
     "@types/koa-mount": "^4.0.2",
     "@types/koa-static": "^4.0.1",
-    "@types/parse5": "^6.0.1",
```

Comment:
Ah – can this be removed because the types come from `parse5` directly?
Very nice!